### PR TITLE
Increase the URL length on API and Samplify

### PR DESCRIPTION
**Pull request by ssi-hu-janos-pelsoczi**
set max url length to 2000

### CODE REVIEW

**Review by andyshipman — APPROVED** [review]
Looks good to me

File: SamplifyAPIClient/LineItem.cs [hunk 1]
```diff
@@ -325,41 +326,41 @@ public class DetailedLineItemReport
...
+        [DataMember(Name = "completesRefusedPercentage")]
+        public decimal CompletesRefusedPercentage { get; set; }
+        [DataMember(Name = "incompletes")]
+        public int Incompletes { get; set; }
+        [DataMember(Name = "incompletesPercentage")]
+        public decimal IncompletesPercentage { get; set; }
+        [DataMember(Name = "lastAcceptedIncidenceRate", IsRequired = false)]
+        public decimal LastAcceptedIncidenceRate { get; set; }
+        [DataMember(Name = "lastAcceptedLOI", IsRequired = false)]
+        public int LastAcceptedLOI { get; set; }
```
Comment (caseylucas) [inline SamplifyAPIClient/LineItem.cs]: This change appears to be unrelated to url length. Unless this change is related, can you remove it?

File: SamplifyAPIClient/Project.cs [hunk 2]
```diff
@@ -1,53 +1,53 @@
...
+        [DataMember(Name = "billingID")]
+        public string ID { get; set; }
+        [DataMember(Name = "type")]
+        public string BillingType { get; set; }
+        [DataMember(Name = "billingDate")]
+        public string RawDTStringBillingDate { get; set; }
+
+        [IgnoreDataMember]
+        public DateTime? BillingDate => Util.ConvertToDateTimeNullable(RawDTStringBillingDate);
+    }
```
Comment (caseylucas) [inline SamplifyAPIClient/Project.cs]: Remove unrelated change.

**Review by caseylucas — CHANGES_REQUESTED** [review]
see comments

File: SamplifyAPIClient/Project.cs (hunk 2 above)
Comment (ssi-hu-janos-pelsoczi) [reply SamplifyAPIClient/Project.cs]: Not changed this part, only change line 181: Validator.ValidateUrlLength(this.SurveyURL);

**Review by caseylucas — APPROVED** [review]
@ssi-hu-janos-pelsoczi I put back the original line endings. If you agree with the changes as they stand, please merge PR.

**Pull request merged**